storage/libradb/src/system_store/test.rs

```diff
@@ -40,38 +58,35 @@ fn test_inc_ledger_counters() {
     }
     // Second batch, add to the first batch.
     {
-        let mut bumps = LedgerCounterBumps::new();
-        bumps.bump(LedgerCounter::EventsCreated, 20);
+        let bumps = create_bumps_map(2, vec![5; 9]);
 
         let counters = bump_ledger_counters(
             store, 2,  /* first_version */
             10, /* last_version */
             bumps,
         );
-        assert_eq!(counters.get(LedgerCounter::EventsCreated), 30);
+        assert_eq!(counters.get(LedgerCounter::EventsCreated), 55);
     }
-    // Add to an old version.
+    // Add to an old version
     {
-        let mut bumps = LedgerCounterBumps::new();
-        bumps.bump(LedgerCounter::EventsCreated, 5);
+        let bumps = create_bumps_map(4, vec![1, 2, 3, 4, 5]);
 
         let counters = bump_ledger_counters(
-            store, 2, /* first_version */
+            store, 4, /* first_version */
             8, /* last_version */
             bumps,
         );
-        assert_eq!(counters.get(LedgerCounter::EventsCreated), 15);
+        assert_eq!(counters.get(LedgerCounter::EventsCreated), 35);
     }
-    // Base version missing, swallowing the error and adding to zeros.
+    // Base version and some entries are missing, swallowing the error and adding to zeros.
```

Comment:
could you add a test to show that if a counter at a version doesn't bump, it retains the same value as the last version?